packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash/sqm-tax-and-cash.tsx

```diff
@@ -0,0 +1,1041 @@
+   * @uiName QST Tax Number input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_qstNumber: string = "QST Number";
+  /**
+   * Label text for the QST checkbox
+   * @uiName checkbox label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isRegisteredQST: string = "I am registered for QST Tax";
+  /**
+   * Label text for Income Tax checkbox
+   * @uiName Income Tax checkbox label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isRegisteredSubRegionIncomeTax: string =
+    "I am an individual registered for Income Tax purposes in Spain, and withholding tax will apply to any payments made to me.";
+  /**
+   * Label text for Sub-region select
+   * @uiName for Sub-region select label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_subRegion: string = "Sub-region";
+  /**
+   * Label text for the Sub-region Income Tax Number input
+   * @uiName Sub-region income Tax Number input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_subRegionTaxNumberLabel: string = "Income Tax Number";
+  /**
+   * Alert header text shown in alert if user is already a registered partner
+   * @uiName Participant is partner title
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isPartnerAlertHeader: string =
+    "An account with this email already exists with our referral program provider, impact.com";
+
+  /**
+   * Alert description text shown in alert if user is already a registered partner
+   * @uiName Participant is partner description
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isPartnerAlertDescription: string =
+    "If you don’t recognize this referral program provider or believe this is a mistake, please contact Support or sign up for this referral program with a different email.";
+
+  /**
+   * Text shown inside of submit button
+   * @uiName Submit button text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_submitButton: string = "Continue";
+  /**
+   * Text shown inside of back button
+   * @uiName Back button text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_backButton: string = "Back";
+  /**
+   * Error text shown below the tax details radio buttons
+   * @uiName Indirect tax details error text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_taxDetailsError: string = "This field is required";
+  /**
+   * Alert text indicating participant cannot change info after it has been submitted
+   * @uiName Cannot change info Alert text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_cannotChangeInfoAlert: string =
+    "Changes to your personal information and indirect tax can only be made through our Support team after you complete this step. Ensure these are correct before continuing.";
+
+  /**
+   * Error text shown below the Selected Region select input
+   * @uiName Selected Region error text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_selectedRegionError: string = "Country is required";
```

Comment:
This and the prop below can be condensed into the general {fieldName} is required prop too